src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -23,6 +23,14 @@
 /** A candidate for a public office position that will appear on voter ballots */
 @AutoValue
 public abstract class Candidate {
+  public static final String ENTITY_NAME = "Candidate";
+  public static final String NAME_JSON_KEYWORD = "name";
+  public static final String PARTY_JSON_KEYWORD = "party";
+  public static final String NAME_OBJECT_KEYWORD = "name";
```

Comment:
Hm, should we replace this with "ENTITY"? since they should be used in from/to Entity related methods.